packages/wallet-service/package.json

```
@@ -40,12 +40,13 @@
     "winston": "3.13.0"
   },
   "peerDependencies": {
-    "@hathor/wallet-lib": "0.39.0",
+    "@hathor/wallet-lib": "1.14.1",
     "@wallet-service/common": "1.5.0"
   },
   "devDependencies": {
     "@types/aws-lambda": "8.10.95",
     "@types/jest": "^29.5.13",
+    "@types/joi": "^17.2.3",
```

```suggestion
    "@types/joi": "17.2.3",
```

Do the same for the above library

---

Done